src/http/server/handlers/static_file_handler.cr

```diff
@@ -6,6 +6,12 @@ require "uri"
 class HTTP::StaticFileHandler
   include HTTP::Handler
 
+  ALLOWED_METHODS        = {"GET", "HEAD"}
+  HTTP_IF_MODIFIED_SINCE = "If-Modified-Since"
+  HTTP_IF_NONE_MATCH     = "If-None-Match"
+  HTTP_ETAG              = "Etag"
+  HTTP_LAST_MODIFIED     = "Last-Modified"
```

Comment:
these consts should be private, or maybe even just removed and inlined. We don't typically agressively extract literals in crystal.